core/tests/linter_tests/invalid_explore.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
```

Comment:
Unable to understand the useage of this file with the given filename*

Reply:
This file contain `browsed.explore()` which is prohibited to use in the codebase.

Reply:
Done.